lift/Readme.md

```diff
@@ -1,13 +1,15 @@
-🚨 **NOTE: This spec is outdated**
+# Lift
 
-- Start Date: 2019-03-22
-- RFC PR: (leave this empty)
-- Prisma Issue: (leave this empty)
-
-# Summary
+- Owner: @schickling
```

Comment:
Why is @schickling the owner here? Why is not @timsuchanek anymore?

Reply:
Because Tim is not part of the product org, which is the usual pool we choose from for "Owner". He doesn't drive the spec part, he drives the implementation part.

(Although currently @matthewmueller is reworking the spec, so I will add him as an additional stakeholder)